libs/types/src/cdf/ballot-definition/convert.ts

```diff
@@ -45,6 +58,174 @@ export function convertVxfElectionToCdfBallotDefinition(
   const stateId = vxfElection.state.toLowerCase().replaceAll(' ', '-');
   const electionDate = dateString(new Date(vxfElection.date));
 
+  const precinctSplits: Record<
+    Vxf.PrecinctId,
+    | Array<{
+        split: Cdf.ReportingUnit;
+        ballotStyles: Vxf.BallotStyle[];
+      }>
+    | undefined
+  > = Object.fromEntries(
+    vxfElection.precincts.map((precinct) => {
+      const precinctBallotStyles = vxfElection.ballotStyles.filter(
+        (ballotStyle) => ballotStyle.precincts.includes(precinct.id)
+      );
+      // There may be multiple ballot styles with the same districts but
+      // different partyIds, but we only want to split precincts that are part
+      // of different districts.
```

Comment:
So this comment is basically noting that there's a second reason why a single precinct might have multiple ballot styles. The first reason is a split precinct. The second reason is ballot styles for different party affiliations.

Reply:
Yup! This is used for primary elections. VxDesign doesn't actually generate the separate ballot styles for party affiliations yet (it's next on the roadmap), but we have fixtures that already have party-affiliated ballot styles.